internal/scripts/execute.go

```diff
@@ -60,6 +60,8 @@ var ExecuteCommand = &command.Command{
 			return nil, fmt.Errorf("provide a valide filename command argument")
 		}
 
+		scriptFlags.Arg, _ = cmd.Flags().GetStringArray("arg")
```

Comment:
Running this with same flag syntax seems to produce error: `trying to get stringArray value of flag of type stringSlice`